learning_wavelets/training_scripts/exact_recon_unet_training.py

```diff
@@ -51,7 +53,7 @@ def train_unet(
 
     # callbacks preparation
 
-    chkpt_cback = ModelCheckpoint(chkpt_path, period=n_epochs, save_weights_only=True)
+    chkpt_cback = ModelCheckpoint(chkpt_path, period=500, save_weights_only=True)
```

Comment:
```suggestion
    chkpt_cback = ModelCheckpoint(chkpt_path, period=min(500, n_epochs), save_weights_only=True)
```

This way you won't do training that don't save their weights